app/serializers/important_events_serializer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ImportantEventsSerializer
+  include JSONAPI::Serializer
+
+  set_type :events
+
+  attributes :event_type, :classification
```

Comment:
@solidgoldpig Any critical or medical events will be serialized with just these minimal attributes; enough to build a list of flags in the UI hopefully.

Reply:
Will the timeline events also have the classification?

Reply:
Yes, I've added classification to those too 👍